### PR TITLE
Better user notification UI

### LINKED ISSUE #478: Displaying notification messages causes the page content to jump
When notification messages (Such as article creation success/failure) are displayed and removed automatically after a few seconds one can notice a jump in the page content.

### PR DESCRIPTION
**Pull request by breezyboa**
Fixes #478

### CODE REVIEW

**Comment by breezyboa** [comment]
@tejaswinichile _a can you review this

**Comment by tejaswinichile** [comment]
@breezyboa How are we hiding flash messages as we have removed timeout?

**Comment by breezyboa** [comment]
@tejaswinichile it is handled in `UserNotification` module in the update method using Process.sleep

File: app/javascript/Admin/Page/UserNotification.elm [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    ( initModel, [] )
+
+
+
+-- UPDATE
+
+
+type Msg
+    = InsertNotification UserNotification
+    | RemoveNotification UserNotification
```
Comment (tejaswinichile) [inline app/javascript/Admin/Page/UserNotification.elm]: I am not able to understand the definition for this, can you add the difference between `ClearNotification` and `RemoveNotification`?

File: app/javascript/Admin/Page/UserNotification.elm [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            )
+
+        RemoveNotification notification ->
+            ( { model | notifications = List.filter (not << isNotificationEqual notification) model.notifications }, [] )
+
+        ClearNotifications ->
+            ( { model | notifications = [] }, [] )
+
+
+isNotificationEqual notificationA notificationB =
```
Comment (tejaswinichile) [inline app/javascript/Admin/Page/UserNotification.elm]: `notificationA`, `notificationB` ?

**Comment by tejaswinichile** [comment]
@breezyboa _a

File: app/javascript/Admin/Page/UserNotification.elm (hunk 1 above)
Comment (breezyboa) [reply app/javascript/Admin/Page/UserNotification.elm]: `RemoveNotification` removes a single/specific notification from the list
`ClearNotifications ` clears all notifications.

File: app/javascript/Admin/Page/UserNotification.elm (hunk 2 above)
Comment (breezyboa) [reply app/javascript/Admin/Page/UserNotification.elm]: This functions takes 2 `UserNotification ` s, `notificationA` and  `notificationB` and compares it to see if they are equal.

**Comment by breezyboa** [comment]
@tejaswinichile _a

**Comment by tejaswinichile** [comment]
@breezyboa _a Please resolve conflicts.

**Comment by breezyboa** [comment]
@tejaswinichile _a

**Pull request merged**